src/drivers/distance_sensor/tfmini/tfmini_parser.h

```diff
@@ -66,7 +94,11 @@ enum class TFMINI_PARSE_STATE {
 	STATE3_GOT_STRENGTH_H,
 	STATE4_GOT_RESERVED,
 	STATE5_GOT_QUALITY,
-	STATE6_GOT_CHECKSUM
+	STATE6_GOT_CHECKSUM,
+	STATE7_GOT_COMMANDRESPONSE,
```

Comment:
Would you be willing to change this and do likewise in the next line to `STATE7_GOT_COMMAND_RESPONSE` to make it a little easier to read?